src/io/IOHelpers.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "molecules/Component.h"
+#include "utils/Random.h"
+
+class ParticleContainer;
+class DomainDecompBase;
+
+namespace IOHelpers {
+
+void removeMomentum(ParticleContainer* particleContainer, const std::vector<Component>& components,
+					DomainDecompBase* domainDecomp);
+
+void initializeVelocityAccordingToTemperature(ParticleContainer* particleContainer, DomainDecompBase* domainDecomp,
+											  double temperature);
+
+unsigned long makeParticleIdsUniqueAndGetTotalNumParticles(ParticleContainer* particleContainer,
+														   DomainDecompBase* domainDecomp);
+
+std::array<double, 3> getRandomVelocityAccordingToTemperature(double temperature, Random& RNG);
```

Comment:
add documentation

Reply:
added doc in https://github.com/ls1mardyn/ls1-mardyn/pull/175/commits/a2f46aec53f89fb5450a0aa5bb44c0e9aec2c56c